dojo/tools/mobsf/parser.py

```diff
@@ -372,11 +372,10 @@ def getSeverityForPermission(self, status):
     # Criticality rating
     def getCriticalityRating(self, rating):
         criticality = "Info"
-        if rating == "warning":
+        if rating == "Good":
```

Comment:
```suggestion
        if rating in ["warning", "Good"]:
```

Reply:
@Maffooch that "warning" here is useless as it is always replaced already:
https://github.com/DefectDojo/django-DefectDojo/blob/eb3cf16bc68f39069df8154eba11e236130b9a6e/dojo/tools/mobsf/parser.py#L137
https://github.com/DefectDojo/django-DefectDojo/blob/eb3cf16bc68f39069df8154eba11e236130b9a6e/dojo/tools/mobsf/parser.py#L146
https://github.com/DefectDojo/django-DefectDojo/blob/eb3cf16bc68f39069df8154eba11e236130b9a6e/dojo/tools/mobsf/parser.py#L162
https://github.com/DefectDojo/django-DefectDojo/blob/eb3cf16bc68f39069df8154eba11e236130b9a6e/dojo/tools/mobsf/parser.py#L172

warning at this place does basically nothing, I have removed it entirely and all unittests passed